Dockerfile

```diff
@@ -1,22 +1,31 @@
-FROM golang:1.11.4-alpine3.8 AS builder
-LABEL maintainer="joona@kuori.org"
+ARG ALPINE_VERSION=3.9
+
+FROM golang:1.12-alpine${ALPINE_VERSION} AS builder
+
+ARG ACME_DNS_VERSION=v0.7.2
+
+WORKDIR /go/src/github.com/joohoi
 
-RUN apk add --update gcc musl-dev git
+RUN apk add --no-cache gcc musl-dev git && \
+    git clone -b ${ACME_DNS_VERSION} --depth 1 https://github.com/joohoi/acme-dns.git acme-dns
 
-RUN go get github.com/joohoi/acme-dns
 WORKDIR /go/src/github.com/joohoi/acme-dns
+
 RUN CGO_ENABLED=1 go build
 
-FROM alpine:latest
+RUN install -d -o 1000 -g 0 -m 0775 /build/etc/acme-dns /build/var/lib/acme-dns && \
+    install -D -o 1000 -g 0 -m 0554 /go/src/github.com/joohoi/acme-dns/acme-dns /build/usr/local/bin/acme-dns && \
+    install -D -o 1000 -g 0 -m 0664 /go/src/github.com/joohoi/acme-dns/config.cfg /build/etc/acme-dns/config.cfg && \
+    sed -Ei -e "s/(port ?= ?)\"80\"/\1\"8080\"/" \
+            -e "s/(listen ?= ?)\".*:53\"/\1\":5353\"/" /build/etc/acme-dns/config.cfg
+
 
-WORKDIR /root/
-COPY --from=builder /go/src/github.com/joohoi/acme-dns .
-RUN mkdir -p /etc/acme-dns
-RUN mkdir -p /var/lib/acme-dns
-RUN rm -rf ./config.cfg
-RUN apk --no-cache add ca-certificates && update-ca-certificates
```

Comment:
Please restore the line to add "`ca-certificates`". Removing them from the final docker image, breaks the "letsencrypt" TLS option since the ACME client can no longer access the letsencrypt https API.

Reply:
Thanks for your input. I will add it again.